NTC-Weather-Forecast-COVID-19/NTC-Weather-Forecast-COVID-19/Controller/Weather Forecast/SettingTempViewController.swift

```diff
@@ -8,8 +8,57 @@
 import UIKit
 
 final class SettingTempViewController: UIViewController {
+    @IBOutlet private weak var settingTempTableView: UITableView!
 
+    private var currentIndexTempFormat = 0
+    private let identifierCell = "SettingTempTableViewCell"
+    private let keyTempFormat = UserDefaultsKeys.keyTempFomat.rawValue
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        settingTempTableView.delegate = self
+        settingTempTableView.dataSource = self
+        loadTempFormatFromUserDefaults()
+    }
+    
+    private func updateTempFormatInUserDefaults(newValue: Int) {
+        UserDefaults.standard.set(newValue, forKey: keyTempFormat)
+    }
+    
+    private func loadTempFormatFromUserDefaults() {
+        if let indexTempFormat = UserDefaults.standard.value(forKey: keyTempFormat) as? Int {
+            currentIndexTempFormat = indexTempFormat
+        } else {
+            currentIndexTempFormat = 0
+        }
+    }
```

Comment:
add thẳng vào viewDidLoad
```suggestion
        currentIndexTempFormat = (UserDefaults.standard.value(forKey: keyTempFormat) as? Int) ?? 0
```

Reply:
vâng ạ